addons/LPCAnimatedSprite/LPCAnimatedSprite2D.gd

```diff
@@ -147,8 +162,11 @@ func AddAnimation(spriteSheet:Texture, spriteFrames:SpriteFrames, animationData:
 		if "WALK" in animationData.Name && col == 0:
 			continue
 		var atlasTexture = AtlasTexture.new()
-		atlasTexture.atlas = spriteSheet
-		atlasTexture.region = Rect2(64*col,64*animationData.Row,64,64)
+		atlasTexture.atlas = spriteSheet.SpriteSheet
+		var spriteSize:int = 64
+		if spriteSheet is OversizedLPCSpriteSheet:
+			spriteSize = 192
+		atlasTexture.region = Rect2(spriteSize*col,spriteSize*animationData.Row,spriteSize,spriteSize)
```

Comment:
I really like this way of doing it. Did you give any thought to the idea of a starting column? It would be great to provide offset starting columns (like the "HURT_DOWN" last frame of the incapacitated person idea I mentioned.)

Reply:
like a static frame same as idle?

Reply:
It would require a column as well though, right? IDLE_X all work that way but all begin in the first column. The last frame of HURT_DOWN would require starting on another column.

Reply:
Understood